src/index.js

```diff
@@ -453,9 +453,6 @@ function bindAll(ctx) {
 
 
 function callMethod(ctx, m, args) {
-	if (typeof m==='string') {
```

Comment:
Is this change necessary for the PR?  If not please remove it and submit as another PR - at first glance, it breaks libraries that reassign to `this.render` after instantiation.